mkdocs.yml

```diff
@@ -313,6 +313,13 @@ plugins:
         "models/extras/objectmetadata.md": "user-guide/platform-functionality/objectmetadata.md"
         "models/extras/savedview.md": "user-guide/platform-functionality/savedview.md"
         "models/extras/staticgroupassociation.md": "user-guide/platform-functionality/staticgroupassociation.md"
+        "models/wireless/accesspointgroup.md": "user-guide/core-data-model/wireless/accesspointgroup.md"
+        "models/wireless/supporteddatarate.md": "user-guide/core-data-model/wireless/supporteddatarate.md"
+        "models/wireless/radioprofile.md": "user-guide/core-data-model/wireless/radioprofile.md"
+        "models/wireless/wirelessnetwork.md": "user-guide/core-data-model/wireless/wirelessnetwork.md"
+        "models/wireless/accesspointgroupdeviceassignement.md": "user-guide/core-data-model/wireless/accesspointgroupdeviceassignment.md"
+        "models/wireless/accesspointgroupradioprofileassignment.md": "user-guide/core-data-model/wireless/accesspointgroupradioprofileassignment.md"
+        "models/wireless/accesspointgroupwirelessnetworkassignement.md": "user-guide/core-data-model/wireless/accesspointgroupwirelessnetworkassignment.md"
```

Comment:
If these through tables aren't going to have significant/substantial data or behavior of their own, I'd suggest rolling them into the documentation of their relevant parent models instead. But that's not something we've been consistent about thus far, I admit.

Reply:
The tests will likely assume that these have docs through the get_docs_url test. I stubbed these out for the test, but I will follow the same documentation that we did for cloud models.

Reply:
I *think* the tests will work so long as we have appropriate redirects in place.